appsec-rules/crowdsecurity/vpatch-CVE-2022-1388.yaml

```diff
@@ -0,0 +1,30 @@
+## autogenerated on 2025-04-23 16:03:44
+name: crowdsecurity/vpatch-CVE-2022-1388
+description: 'Detects F5 BIG-IP iControl REST authentication bypass and RCE via crafted POST to /mgmt/tm/util/bash with X-F5-Auth-Token header.'
+rules:
+  - and:
+      - zones:
+          - URI
+        transform:
+          - lowercase
+        match:
+          type: contains
+          value: /mgmt/tm/util/bash
+      - zones:
+          - HEADERS_NAMES
+        transform:
+          - lowercase
+        match:
+          type: contains
+          value: x-f5-auth-token
+labels:
+  type: exploit
+  service: http
+  confidence: 3
+  spoofable: 0
+  behavior: 'http:exploit'
+  label: 'F5 BIG-IP iControl REST Auth Bypass RCE'
```

Comment:
I think it should be something like : "F5 BIG-IP iControl REST - Authentication Bypass" ? It seems that the RCE is just another functionality / bug ?